src/layouts/MainLayout.astro

```diff
@@ -9,7 +9,7 @@ import * as CONFIG from "../config"
 import { NewsletterCTA } from "../components/Footer/NewsletterCTA"
 import Footer from "~/components/Footer/Footer.astro"
 
-const { content = {}, frontmatter } = Astro.props
+const { frontmatter, frontmatter: content = {} } = Astro.props
```

Comment:
Question:

Why are we destructuring twice the same property? Is this intended or is this a typo?

Reply:
I am not sure why, but before this PR, it seems that the `content` object was used as a `frontmatter` in the MainLayout. So after I moved all the pages to src/content, I had to assign the `content` to the frontmatter value, so that the MainLayout would work as intended. 
I have modified the MainLayout by removing the 'content' variable and replacing it with `frontmatter` instead. Seems like it works without any issues as before 🤷‍♀️ 

However, I am not sure why the `content` object was used in this case. I tried to search in Astro Docs, but I couldn't find the answer.